### DISCUSSION TITLE
Icon contribution for Big Sur on next release

### DISCUSSION THREAD

**Discussion by jsejcksn** [pilotmoon/Scroll-Reverser#112] [Show and tell]
Hello @invariant

I don't know the history of your iconography (10+ years, right?), and I know design can be very personal to an individual. With that context acknowledged, I want to contribute a concept for an updated app icon for Big Sur on the next release. Here are a few screenshots comparing current and proposed, as well as the source design file (exported from Figma), which can be iterated on by you or other contributors.

<img width="300" alt="Current icon" title="Current icon" src="https://user-images.githubusercontent.com/868251/106852663-0aeabb00-667e-11eb-85b2-bc7308a66659.png"> <img width="300" alt="Proposed icon"  title="Proposed icon" src="https://user-images.githubusercontent.com/868251/106852683-16d67d00-667e-11eb-831e-5b7856d10870.png">

<img width="300" alt="current" title="current" src="https://user-images.githubusercontent.com/868251/106852745-2eae0100-667e-11eb-910f-2d300329ef04.png"> <img width="300" alt="proposed" title="proposed" src="https://user-images.githubusercontent.com/868251/106852759-3372b500-667e-11eb-808e-60f8b450e952.png">

[ScrollReverser.fig.zip](https://github.com/pilotmoon/Scroll-Reverser/files/5923613/ScrollReverser.fig.zip)

**Comment by pilotmoon** [comment]
Thanks a lot! I have had a fair few comments about the icon design now. I'm quite attached to the original but a lot of people seem to get triggered by icons that aren't in the Big Sur compliant style. Personally the Big Sur icon style (not yours specifically but the general concept of everything having a roundrect) is not to my taste, and I'm minded to keep the old one for now but we'll see....! All the same I appreciate this contribution.

(What I do like in the new guidelines is that icons are supposed to be flat face on and not angled jauntily. I think that is an improvemenb and all my apps already conform to that aspect so they don't look so out of date as they might. Just my opinion anyway.)

**Comment by pilotmoon** [comment]
Hey, thanks again for this. In the end I went with a different design (#169) someone else submitted as a pull request (it made it easy for me to merge with one click...). So, I'll close this discussion now.
